src/error.rs

```diff
@@ -203,7 +206,9 @@ impl Error {
             Error::RequiredEkuNotFound => 24,
             Error::NameConstraintViolation => 23,
             Error::PathLenConstraintViolated => 22,
-            Error::CaUsedAsEndEntity | Error::EndEntityUsedAsCa => 21,
+            Error::CaUsedAsEndEntity
+            | Error::EndEntityUsedAsCa
+            | Error::MaximumSignatureChecksExceeded => 21,
```

Comment:
not immediately clear why this error has the same rank as the other two here -- but in practice the new error is a special case and never subject to ranking, right?

Reply:
> in practice the new error is a special case and never subject to ranking, right?

Yeah :+1: 

I moved the new error down to the bottom w/ rank 0 and called out that it's a special case.